Dockerfile

```diff
@@ -4,7 +4,7 @@ ARG ODKLITE_TAG=latest
 FROM obolibrary/odklite:${ODKLITE_TAG}
 LABEL maintainer="obo-tools@googlegroups.com"
 
-ENV PATH="/tools/apache-jena/bin:/tools/sparqlprog/bin:$PATH"
+ENV PATH "/tools/apache-jena/bin:/usr/local/share/swi-prolog/pack/sparqlprog/bin:$PATH"
```

Comment:
https://docs.docker.com/reference/build-checks/legacy-key-value-format/

😂

Reply:
Doh!

So what probably happened is that I started fixing this issue while I was on the 1.5 maintenance branch (where your fix for the `ENV` declaration is not present), and then when I realized that I switched over to the master branch -- and in the process, I ported back to the master branch the old `ENV` declaration.

My bad.